manifests/repo.pp

```diff
@@ -56,17 +58,17 @@
         }
       }
 
-      rpmkey { '3672BA8F':
+      rpmkey { '4021833E14CB7A8D':
```

Comment:
Can an `rpmkey` resource use the full fingerprint as its title? If so it might be better to consistently use the full id :wink:.

Reply:
Doesn't look that way:
> The name of the rpmkey resource has to be the keyID of the gpg key.

Reply:
Marking as resolved as rpmkey needs an ID and apt complains if you don't use a full fingerprint.